### PR TITLE
[FEATURE] Introduce site sets for v13

### PR DESCRIPTION
**Pull request by linawolf**
See also https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/SiteHandling/SiteSets.html

SiteSets

Resolves https://github.com/georgringer/news/issues/2475

### CODE REVIEW

**Comment by linawolf** [comment]
Depends on #2520

File: Configuration/TypoScript/SeoSitemap/setup.typoscript [hunk 1]
```diff
@@ -17,10 +17,10 @@ plugin.tx_seo {
 
 						sortField = datetime
 						lastModifiedField = tstamp
-						pid = {$plugin.tx_news.settings.sitemap.startingpoint}
-						recursive = {$plugin.tx_news.settings.sitemap.recursive}
+						pid = {$plugin.tx_news.settings.sitemap.startingpoint ?? $news.seo.sitemap.startingpoint}
+						recursive = {$plugin.tx_news.settings.sitemap.recursive ?? news.seo.sitemap.recursive}
```
Comment (chrissonntag) [inline Configuration/TypoScript/SeoSitemap/setup.typoscript:21]: Hi there :) I'm just wondering what syntax is actual valid?
`?? $news...` or `?? news...`
With or without $ on the right right of the ??-operator. Or are both valid?

Comment (chesio) [reply Configuration/TypoScript/SeoSitemap/setup.typoscript:21]: The variant with `$` on the right side of `??` operator seems to be the right way: https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/Configuration/TypoScript/Syntax/Operators/Index.html#null-coalescing-operator-for-typoscript-constants

Comment (chrissonntag) [reply Configuration/TypoScript/SeoSitemap/setup.typoscript:21]: Thanks for clarifying

**Pull request closed without merging**